lib/openstax/aws/image.rb

```diff
@@ -0,0 +1,26 @@
+module OpenStax::Aws
+  class Image
+
+    attr_reader :image
```

Comment:
Can you rename to `aws_image` so that when someone has an `image` object they can say `image.aws_image` and know that they are getting an AWS type?